packages/api/src/plugin/Plugins.ts

```diff
@@ -1,10 +1,12 @@
 import { InjectableClass, InjectableFunction, InjectionToken } from 'typed-inject';
+
 import { ConfigEditor } from '../../config';
 import { Mutator } from '../../mutant';
 import { Reporter } from '../../report';
 import { TestFramework } from '../../test_framework';
 import { TestRunner } from '../../test_runner';
 import { Transpiler } from '../../transpile';
+
 import { PluginContexts } from './Contexts';
```

Comment:
Why are these paths in a different block than the relative paths above?

Reply:
The path is '../../' and './' so it's separated (I am not really able to configure autofix for each case, but I guess it isn't that big of a problem here :/ (since it still makes sense))